### ISSUE TITLE
Add documentation for loading collections and initializing waterline

### ISSUE THREAD

**Issue by devinivy** [balderdashy/waterline-docs#52]
Adapters and connections configuration info is missing here.  Also need to document how to extend and load model definitions.  Basically, we need to document how to initialize waterline, drawing all the necessary connections between adapters, connections, and model definitions.

**Comment by dmarcelino** [comment]
Agreed, some of that info already exists in "code format" in [waterline example](https://github.com/balderdashy/waterline/tree/master/example) which can be used as a base for the docs.

**Comment by tribou** [comment]
:+1: Tried reading through the waterline example, and formal documentation would definitely help onboard new users.  Ironically, @devinivy's example in the [waterline-fixtures](https://github.com/devinivy/waterline-fixtures) package README helped the most for me.

Docs relating waterline functions to formal database migrations used for testing would be helpful as well.  e.g. An example dropping/rebuilding all models before a test.
